src/content/docs/query-your-data/explore-query-data/use-charts/nrql-predictions.mdx

```diff
@@ -0,0 +1,133 @@
+
+## Enrollment [#enrollment]
+To enroll for the <DNT>**NRQL Predictions**</DNT> feature, navigate to <DNT>**[one.newrelic.com > Administration](https://one.newrelic.com//admin-portal) > Preview & Trials**</DNT>, and enable <DNT>**NRQL predictions**</DNT>.
+
+## Add prediction on a line or area chart [#add-prediction]
+To add prediction on a line or area chart, follow these steps:
+
+<Steps>
+
+  <Step>
+    ### Access a line or area chart [#access-a-chart]
+
+    1. Go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities).
+    2. To add prediction to a line or area chart, select it from <DNT>**Dashboards**</DNT>, or open it from <DNT>**Metrics and events**</DNT>.
+  </Step>
+
+  <Step>
+    ### Add prediction [#add-prediction]
+
+  To add prediction, from the <Icon name="fe-more-horizontal"/> menu, select <DNT>**Predict trend**</DNT>. In **Data Explorer**, the prediction trend appears on the chart, projecting future trends. The prediction range is automatically set to 20% of the query window, and highlighted in gray. The graph appears in dotted lines within the predicted range. 
+
+  The chart query populates with the `PREDICT` function. You can [refine the query](/docs/nrql/nrql-syntax-clauses-functions/#sel-prediction) according to your requirements and run it to get the updated trend.
+
+  </Step>
+
+  <Step>
+    ### Add to dashboard [#add-to-dashboard]
+
+    1. To add the chart with prediction to a dashboard, click <DNT>**Add to dashboard**</DNT>.
+    2. Enter a widget name.
+    3. Select a dashboard in which you want to add the widget, or create a new dashboard for it.
+
+    You can now view the chart with prediction on the selected dashboard.
+
+  </Step>
+  
+</Steps>
+
+## Default prediction range [#default-prediction-range]
+When you add a prediction to a chart, the `PREDICT` function appends to the existing chart query with the following default behaviors.
+* **Prediction range**: Projects 20% of the total time specified in your query window, allowing you to see a short-term projection based on current trends.
+* **Historical data**: Processes historical data from the current query window and the two preceding ones to generate predictions.
+* **Time interval**: Matches the data point interval of the prediction on the query window, ensuring consistency in data projection.
+
+Example:
+
+```sql
+FROM CrossClusterQuery SELECT count(*) WHERE status NOT IN ('successful', 'rejected') TIMESERIES PREDICT
```

Comment:
CrossClusterQuery is an internal monitoring event that's not accessible to customers. Let's change this to "FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES PREDICT"